components/omega/src/drivers/standalone/OceanDriver.cpp

```diff
@@ -0,0 +1,61 @@
+//===-- drivers/standalone/OceanDriver.cpp - Standalone driver --*- C++ -*-===//
+//
+//
+//===----------------------------------------------------------------------===//
+
+#include "OceanDriver.h"
+#include "DataTypes.h"
+#include "OceanState.h"
+#include "OmegaKokkos.h"
+#include "TimeMgr.h"
+
+#include "mpi.h"
+
+#include <iostream>
+
+int main(int argc, char **argv) {
+
+   int ErrAll;
+   int Err1;
+   int Err2;
```

Comment:
Maybe the following  names would be better ?
```suggestion
   int ErrCurr;
   int ErrFinalize;
```